consoles/sshVirtsh.pm

```diff
@@ -144,8 +144,10 @@ sub _init_xml ($self, $args = {}) {
         }
         if (!$bmwqemu::vars{BIOS}) {
             # We know this won't go well.
+            # uncoverable block
```

Comment:
Where did you find this? This isn't listed as a way to mark uncoverable lines.
https://metacpan.org/pod/Devel::Cover#UNCOVERABLE-CRITERIA

Reply:
I think is possible to use `# uncoverable branch true` to skip the entire if

Reply:
checking. 
@Wabri `# uncoverable branch true` didnt work(coverage didnt change). I try with `false`. Maybe it needs both?! give me some time

Reply:
Some background: Devel::Cover supports statement and branch coverage.
If we used that for our reports, our coverage percentage would actually drop. We are only using line coverage right now, mainly because that's the only thing that codecov supports. (And I guess supporting branch coverage as a provider for code coverage reporting for multiple languages would be too much effort)

So `# uncoverable branch` is only useful if we actually enable branch coverage.
It's necessary in all cases to add `# uncoverable statement` to each line.
And btw you can just use `make coverage TESTS=t/...` to try it out locally and see the Devel::Cover report. There's no need to wait for codecov every time.

Reply:
I followed the `# uncoverable statement` approach as per @perlpunk's last comment. please review again